gridstatus/pjm.py

```diff
@@ -28,6 +28,9 @@ class PJM(ISOBase):
         "https://www.pjm.com/planning/service-requests/services-request-status"
     )
 
+    # PJM requires retries because the API is flaky
+    retries = 3
```

Comment:
can this be set when initializing the PJM class? like `PJM(retries=3)`? that way user has some flexibility. we can still have a default value

Reply:
Good idea, updated.